docs/guides/external-node/docker-compose-examples/README.md

```diff
@@ -0,0 +1,36 @@
+# Running External node locally
+
+This directory contains two docker compose files for running external nodes, for sepolia testnet and mainnet.
+
+They are prepared to work out-of-the-box, no config changes are needed
+
+By default, the HTTP JSON-RPC API will run on port `3050`, while WS API will run on port `3051`.
+
+**Note: The node will recover from a snapshot on it's first run, this may take up to 10h.**
```

Comment:
Bikeshedding: IMO, formatting entire sentences in bold makes them difficult to read; ditto for other such additions. You can use [a GitHub-specific markdown extension](https://github.com/orgs/community/discussions/16925) for a note box, which IMO reads fine even outside of GitHub:

```markdown
> [!NOTE]
> The node will recover from a snapshot on it's first run, this may take up to 10h. Before the recovery is finished, the API server won't serve any requests.
```

Rendered:

> [!NOTE]
> The node will recover from a snapshot on it's first run, this may take up to 10h. Before the recovery is finished, the API server won't serve any requests.

Reply:
This extension is not rendered in github files viewer :'( 
> [!NOTE] If you need access to historical transaction data, please use recovery from DB dumps (see Advanced setup
> section)

Reply:
I've added this particular file to prettierignore, I'm not very proud of it, but it works :D